mtags/src/main/scala/scala/meta/internal/pc/AutoImports.scala

```diff
@@ -48,18 +54,20 @@ trait AutoImports { this: MetalsGlobal =>
           case Some(pkg)
               if pkg.symbol != rootMirror.EmptyPackage ||
                 pkg.stats.headOption.exists(_.isInstanceOf[Import]) =>
-            val lastImport = pkg.stats
+            val lastImport: (Tree, Boolean) = pkg.stats
               .takeWhile(_.isInstanceOf[Import])
-              .lastOption
-              .getOrElse(pkg.pid)
+              .lastOption // if there are no imports, return true to pad top of import
+              .fold[(Tree, Boolean)]((pkg.pid, true))((tree => (tree, false)))
```

Comment:
Maybe just use separate values for each?
```scala
val lastImportOpt = pkg.stats
              .takeWhile(_.isInstanceOf[Import])
              .lastOption
val noImport = lastImport.isEmpty
val lastImport = lastImportOpt.getOrElse(pkg.pid)

```

Reply:
I suppose that is much easier to read 👍